### PR TITLE
Improvements to perf-test(.py)

### PR DESCRIPTION
**Pull request by tautschnig**

### CODE REVIEW

File: scripts/perf-test/make-tables.sh [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+categories=""
+
+for d in "$@"
+do
+  c=$(echo $d/release/logs-* | tr ' ' '\n' | sed 's/.*logs-//')
+  categories=$(echo $categories $c | tr ' ' '\n' | sort | uniq)
+done
+
+offial_logs="
```
Comment (peterschrammel) [inline scripts/perf-test/make-tables.sh]: official?

File: scripts/perf-test/make-tables.sh [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    if [ -s $d/release/logs-$c/*.bz2 ]
+    then
+      srcs="$srcs $d/release/logs-$c/*.bz2"
+    fi
+  done
+
+  if [ -n "$srcs" ]
+  then
+    # TODO: adjust the path to benchexec
+    python3 ~/Desktop/benchexec.git/bin/table-generator -n $c $srcs
```
Comment (peterschrammel) [inline scripts/perf-test/make-tables.sh]: define at least a variable for that

**Comment by tautschnig** [comment]
@peterschrammel I've addressed your comments.

**Review by thk123 — COMMENTED** [review]
Don't know what I am an owner of in this PR as I've not seen this files before. However, if this could be somehow incorporated into a CI nightly job to run these bench marks and put them somewhere visible, that would be amazing. ￼

**Comment by tautschnig** [comment]
@thk123 You're a CODEOWNER of `scripts/`.

The nightly CI job would be nice to have, but is a question of cost: running the full SV-COMP set costs around 20 USD, I think.

**Comment by tautschnig** [comment]
@lucasccordeiro @marek-trtik Would it be possible to similarly incorporate the GCE scripting you've built into the repository?

**Comment by lucasccordeiro** [comment]
We could try to incorporate those scripts from the Google Cloud Platform. We have talked to @zemanlx and he has offered support to us on this task. However, the Travis machines do not seem to be powerful enough to run those benchmarks. According to the configuration given by @zemanlx, we estimate that it will take about 1 week to get the results. Given that, one alternative is to integrate the scripts and run the SV-COMP benchmarks once a month to check whether we had a performance worsening.

**Comment by tautschnig** [comment]
> Given that, one alternative is to integrate the scripts and run the SV-COMP benchmarks once a month to check whether we had a performance worsening.

I'd be really nice to have the scripts available for everyone to use. How to actually schedule the runs should then be the next question. Despite my affiliation I'd prefer to avoid the vendor lock-in that the current perf-test script has.

**Comment by martin-cs** [comment]
> I'd be really nice to have the scripts available for everyone to use. How to actually schedule the runs should then be the next question. Despite my affiliation I'd prefer to avoid the vendor lock-in that the current perf-test script has.

+1 #whathesaid

**Review by peterschrammel — APPROVED** [review]

**Pull request merged**